app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
```

Comment:
The import statements should be grouped and ordered correctly. According to the checklist, standard library imports should come first, followed by local application imports. Therefore, `import datetime` should be placed before the import from `app.errors` .